planner/core/testdata/plan_suite_in.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "TestIndexHint",
+    "cases": [
+      // simple case
```

Comment:
Does the `json` file support comments?

Reply:
No, so before parse, the comment will be removed.